assets/css/edd-admin.css

```diff
@@ -2431,6 +2420,92 @@ body.download_page_edd-reports {
 	margin-bottom: 20px;
 }
 
+div#post-body-content {
+	display: grid;
+	display: -ms-grid;
+	grid-template-columns: 1fr;
+	grid-gap: 20px;
+	max-width: 1000px;
+}
+
+.postbox [class^="edd-export"] {
+	grid-template-rows: 1fr;
+	max-width: 400px;
+}
+
+.edd-reports-wrapper .postbox {
+	margin-bottom: 0;
+}
+#edd-export-earnings select.edd-select {
+	width: 49%;
+}
+
+span.edd-from-to-wrapper {
+	margin-bottom: 15px;
+}
+
+.edd-export-sales-earnings {
+	grid-template-columns: 1fr;
+}
+
+.edd-from-to-wrapper {
+	margin-right: 0;
+}
+
+#edd-order-export-start,
+#edd-order-export-end,
+#edd-orders-export-start,
+#edd-orders-export-end,
+#edd-taxed-orders-export-start,
+#edd-taxed-orders-export-end,
+#edd-api-requests-export-start,
+#edd-api-requests-export-end,
+#edd-file-download-export-start,
+#edd-file-download-export-end {
+	width: 50%;
+	margin-right: 0px;
+}
+
+.edd-import-export-form {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+}
+
+div.to-and-from-container {
+	height: 50px;
+}
+
+#to-label {
```

Comment:
This should be prefixed with `edd-` and perhaps made a bit more specific to Exports? Also using a class instead of an ID will prevent duplicate ID warnings Chrome sometimes shows in the console.